components/constants/webui_url_constants.cc

```diff
@@ -49,6 +49,9 @@ const char kShieldsPanelURL[] = "chrome://brave-shields.top-chrome";
 const char kShieldsPanelHost[] = "brave-shields.top-chrome";
 const char kCookieListOptInHost[] = "cookie-list-opt-in.top-chrome";
 const char kCookieListOptInURL[] = "chrome://cookie-list-opt-in.top-chrome";
+const char kSpeedreaderPanelURL[] = "chrome://brave-speedreader.top-chrome";
+const char kSpeedreaderPanelHost[] = "brave-speedreader.top-chrome";
+const char kSidebarBookmarksHost[] = "sidebar-bookmarks.top-chrome";
```

Comment:
`kSidebarBookmarksHost` doesn't need to be reintroduced - perhaps a bad rebase?

Reply:
yes, bad rebase